conninfo/src/conninfo-v1.0-wd01.md

```diff
@@ -0,0 +1,238 @@
+
+## 1.3 Normative References
+
+(Reference sources:
+For references to IETF RFCs, use the approved citation formats at:  
+http://docs.oasis-open.org/templates/ietf-rfc-list/ietf-rfc-list.html.  
+For references to W3C Recommendations, use the approved citation formats at:  
+http://docs.oasis-open.org/templates/w3c-recommendations-list/w3c-recommendations-list.html.  
+Remove this note before submitting for publication.)
+
+###### [AMQP-v1.0]
+_OASIS Advanced Message Queuing Protocol (AMQP) Version 1.0 Part 0: Overview_. Edited by Robert Godfrey, David Ingham, and Rafael Schloming. Latest version: http://docs.oasis-open.org/amqp/core/v1.0/amqp-core-overview-v1.0.html.
+###### [RFC2119]
+Bradner, S., "Key words for use in RFCs to Indicate Requirement Levels", BCP 14, RFC 2119, DOI 10.17487/RFC2119, March 1997, http://www.rfc-editor.org/info/rfc2119.
+###### [RFC2616]
+Fielding, R., Gettys, J., Mogul, J., Frystyk, H., Masinter, L., Leach, P., and T. Berners-Lee, "Hypertext Transfer Protocol -- HTTP/1.1", RFC 2616, DOI 10.17487/RFC2616, June 1999, <https://www.rfc-editor.org/info/rfc2616>.
+###### [RFC8174]
+Leiba, B., "Ambiguity of Uppercase vs Lowercase in RFC 2119 Key Words", BCP 14, RFC 8174, DOI 10.17487/RFC8174, May 2017, http://www.rfc-editor.org/info/rfc8174.
+
+## 1.4 Non-Normative References
+
+###### [RFC3552]
+Rescorla, E. and B. Korver, "Guidelines for Writing RFC Text on Security Considerations", BCP 72, RFC 3552, DOI 10.17487/RFC3552, July 2003, https://www.rfc-editor.org/info/rfc3552.
+
+-------
+
+# 2 Overview
+
+This specification recommends a standard way for an AMQP container to reveal information about
+itself and its operating environment to its partner.   This allows a container to expose information
+that would be useful for problem diagnosis, such as process identifiers and software version
+information.
+
+This specification is equally appropriate for AMQP container fulfilling the role of clients or
+servers, or containers supporting a peer-to-peer scenario.
+
+-------
+
+# 3 Definitions
+
+An AMQP host that implements this specification MUST provide the following entry in the properties
+field of the AMQP `open` performative [[AMQP-v1.0](#AMQP-v1.0)]. The key of the entry MUST be the symbol
+`connection-info`. The value of the entry MUST be of type `fields`.
+
+The following key values have defined meaning. An implementation MAY choose to omit a key from the map.
+An implementation MAY also include keys of its own. If an implementation receives a key from its partner
+that it does not recognise, it SHOULD ignore it.
+
+# 3.1 connection-info
+
+`<type name="connection-info" class="restricted" source="fields"/>`
```

Comment:
We need the type codes here. I took the 0x0100-0x01FF suffix range for filters in the filter expressions spec and 0x0200-0x02FF for the event streams spec, e.g.

``` XML
 <type name="event-streams-sql-filter" class="restricted" source="sql-filter" provides="filter">
<descriptor name="amqp:event-streams-sql-filter" code="0x00000000:0x00000201"/> 
</type>
```

Reply:
A restricted of type fields (which itself is just a restriction of map) dose not need a descriptor - it is not a described type